app/components/bali/card/image/component.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= tag.send tag_name, **options do %>
-  <%= tag.figure class: "image #{@figure_class}" do %>
+  <%= tag.figure class: "image column is-12 #{@figure_class}" do %>
```

Comment:
Una duda, `column is-12` es para que la imagen tenga un width de 100%?

Reply:
Sí, para que tenga 100%, hay alguna otra forma?

Reply:
Sí lo necesita, porque el figure que está antes del img tiene ancho de 0, y eso hace que al pasar el ratón encima de la imagen se vea parpadeando

Reply:
existe un is-fullwidth

Reply:
No lo vi en la documentación, deja lo pruebo